src/js/rx/containers/Detail.jsx

```diff
@@ -165,7 +165,8 @@ export class Detail extends React.Component {
   }
 
   openGlossaryModal(term) {
-    const content = glossary.filter((obj) => {
+    const allTerms = glossary.Prescription.concat(glossary.Refill);
+    const content = allTerms.filter((obj) => {
```

Comment:
History needs the modal handling as well. Maybe add a helper that's shared between both.